zenodo/modules/records/jsonschemas/records/base-v1.0.0.json

```diff
@@ -588,10 +588,13 @@
     },
     "custom": {
       "additionalProperties": {
-        "type": [
-          "string",
-          "number"
-        ]
+        "additionalProperties": {
```

Comment:
For defining arrays of string/numbers you need to use the `items` key (see https://json-schema.org/understanding-json-schema/reference/array.html)